bpf/bpf_netdev.c

```diff
@@ -714,14 +714,14 @@ int to_netdev(struct __sk_buff *skb)
 	 * workaround.
 	 */
 	int ret = TC_ACT_OK;
-#if defined(ENABLE_NODEPORT) || defined(ENABLE_MASQUERADE)
-#ifdef ENABLE_NODEPORT
+#if (defined(ENABLE_NODEPORT) && !defined(ENABLE_DSR)) || defined(ENABLE_MASQUERADE)
+# if defined(ENABLE_NODEPORT) && !defined(ENABLE_DSR)
```

Comment:
Did ENABLE_MASQUERADE drop?

Reply:
Noup, it's still present at the line 717.